tutor/specs/components/tag-list.spec.jsx

```diff
@@ -0,0 +1,29 @@
+import TagList from '../../src/components/tag-list';
+
+const ITEMS = [
+  { id: 'a', title: 'Title A' },
+  { id: 'b', title: 'Title B' },
+  { id: 'c', title: 'Title C' },
+];
+
+describe('TagList component', () => {
+  let props;
+
+  beforeEach(() => {
+    props = {
+      items: ITEMS,
+      onRemove: jest.fn(),
+    };
+  });
+
+  it('renders items', () => {
+    expect.snapshot(<TagList {...props} />).toMatchSnapshot();
+  });
+
+  it('fires callback when removed', () => {
+    const list = mount(<TagList {...props} />);
+    list.find('.remove-tag').at(3).simulate('click');
+    expect(props.onRemove).toHaveBeenCalledWith(ITEMS[1]);
+    list.unmount();
+  });
+});
```

Comment:
nice spec, it's always good to have at least a minimal one to make sure it renders